src/ngraph/op/pad.cpp

```diff
@@ -174,3 +174,97 @@ std::shared_ptr<Node> op::Pad::get_default_value() const
     }
     return std::make_shared<op::Broadcast>(input_value(1), get_shape(), axes);
 }
+
+const string op::v1::Pad::type_name{"Pad"};
+
+op::v1::Pad::Pad(const Output<Node>& arg,
+                 const Output<Node>& pads_begin,
```

Comment:
According to spec third input should be optional. And third input is necessary only when pad_mode='constant'.